dev/sg/internal/images/images.go

```diff
@@ -376,7 +377,7 @@ func createAndFillImageRepository(ref *ImageReference, pinTag string) (repo *ima
 		latestTag = findLatestTag(tags)
 	}
 
-	if latestTag == ref.Tag || latestTag == "" {
+	if _, err := semver.NewVersion(latestTag); err == nil && (latestTag == ref.Tag || latestTag == "") {
```

Comment:
Maybe just hardcode `insiders` tag? Given this is our only mutable tag.

Reply:
I'm having a bit of a hard time interpreting this change, is the following right?

```suggestion
	_, semverErr := semver.NewVersion(latestTag)
	isSemverTag := semverErr == nil 
	noUpdateNeeded := latestTag == ref.Tag || latestTag == ""
	if isSemverTag && noUpdateNeeded {
```

Reply:
If this isn't too hard to support it would be nice to keep around, we also generate a variety of other tags that might come in from time to time e.g. tags named after branches etc